src/main/java/com/sinthoras/visualprospecting/mixinplugin/MixinPlugin.java

```diff
@@ -72,11 +76,11 @@ public void acceptTargets(Set<String> myTargets, Set<String> otherTargets) {
         return mixins;
     }
 
-    private boolean loadJar(final String jarNameBeginsWith) {
+    private boolean loadJarOf(final TargetedMod mod) {
         try {
-            File jar = MinecraftURLClassPath.getJarInModPath(jarNameBeginsWith);
```

Comment:
This will skip mods in mods/1.7.10

Better to stay with classpath

Reply:
but my implementation is 1-to-1 in terms of logic from `getJarInModPath`, I too use `walk` to traverse all files

Reply:
the only reason to re-implement is that they don't have case-insensitivity and use only `contains` for check when we want a prefix